openshift-install-powervs

```diff
@@ -1186,7 +1188,21 @@ function setup_poweriaas() {
 # If latest is available in System PATH then use symlink
 #-------------------------------------------------------------------------
 function setup_ibmcloudcli() {
-  CLI_LATEST=$(curl -s https://api.github.com/repos/IBM-Cloud/ibm-cloud-cli-release/releases/latest | grep tag_name | cut -d'"' -f4 | sed 's/v//')
+  TAGS=( $(git -c 'versionsort.suffix=-' ls-remote --exit-code --refs --sort='version:refname' --tags \
+  https://github.com/IBM-Cloud/ibm-cloud-cli-release '*.*.*') )
+
+  OS=$(uname -s); 
+  OS_NAME=$(echo "$OS" | awk '{print tolower($0)}')
+
+  ARCH=$(uname -m)
+  if [ "$ARCH" == "x86_64" ]; then
+  	ARCH="amd64"
+  fi
+
+  OS_ARCH="${OS_NAME}_${ARCH}"
+
+  CLI_LATEST=$(echo ${TAGS[${#TAGS[@]}-1]} | awk -F "/" '{print $3}' | tr -d 'v')
+
```

Comment:
The old way to get CLI_LATEST works fine, why do we need to use the new way to get CLI_LATEST value?

Reply:
Same confusion here.